src/components/DayPicker.jsx

```diff
@@ -52,6 +54,8 @@ const defaultProps = {
   modifiers: {},
   orientation: HORIZONTAL_ORIENTATION,
   withPortal: false,
+  focused: true,
```

Comment:
I don't mind if it deviates from the other components. `hidden` is fine, but the opposite of "focused" is "blurred", and that's not a great name either :-/